tasks/shared.py

```diff
@@ -353,7 +352,7 @@ def build_test_envs() -> str:
     return ""
 
 
-def init_yaml_obj(line_length: int | None = None) -> YAML:
+def init_yaml_obj(line_length: int | None = None):  # noqa: ANN201
```

Comment:
Not a big deal but this could probably be handled by having an import for YAML within an `if TYPE_CHECKING` block.